### PR TITLE
manifest: test proxy setting propagates to remote write

### PR DESCRIPTION
**Pull request by jan--f**
* [ ] I added CHANGELOG entry for this change.
* [ ] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

**Comment by jan--f** [comment]
needs #2269 to succeed.

File: pkg/manifests/manifests_test.go [hunk 1]
```diff
@@ -1299,6 +1303,34 @@ func TestPrometheusK8sRemoteWriteURLs(t *testing.T) {
...
+	p, err := f.PrometheusK8s(
+		&v1.Secret{ObjectMeta: metav1.ObjectMeta{Name: "foo"}},
+		&v1.ConfigMap{ObjectMeta: metav1.ObjectMeta{Name: "foo"}},
+		nil,
+	)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if p.Spec.RemoteWrite[0].URL != "https://test.remotewrite.com/api/write" {
```
Comment (machine424) [inline pkg/manifests/manifests_test.go:1326]: nit: maybe we could use `testify/require`

File: pkg/manifests/manifests_test.go [hunk 2]
```diff
@@ -1299,6 +1303,34 @@ func TestPrometheusK8sRemoteWriteURLs(t *testing.T) {
...
+		t.Errorf("want remote write URL https://test.remotewrite.com/api/write, got %v", p.Spec.RemoteWrite[0].URL)
+	}
+	if p.Spec.RemoteWrite[0].ProxyURL != "http://test.proxy.com" {
```
Comment (machine424) [inline pkg/manifests/manifests_test.go:1329]: nit: maybe we could add a check to `TestPrometheusK8sConfiguration` that no proxy is set by default (who knows).

**Review by machine424 — COMMENTED** [review]
/lgtm
/hold
for some nits, don't hesitate to unhold.

**Comment by jan--f** [comment]
/close
test was addressed in #2269

**Pull request closed without merging**